API.yaml

```diff
@@ -749,14 +899,13 @@ paths:
             application/json:
               schema:
                 type: string
-  /experiments/{expUUID}/outputs/XY/{outputID}/xy:
+  /alpha/experiments/{expUUID}/outputs/timeGraph/{outputID}/navigate/states:
```

Comment:
What does means alpha?

I don't think it's Restful to add endpoints for this.

You cannot use requestBody for GET.